// A simple protobuf message instantiation that
// realizes google::protobuf::Message
// @author vbalan

option optimize_for = SPEED;

message MessageBase {
}